### PR TITLE
Portfolioredux

### PR DESCRIPTION
**Pull request by munibk22**
"Full portfolio Redux"

### CODE REVIEW

File: src/components/Account/LoginValidationHook.tsx [hunk 1]
```diff
@@ -33,14 +33,14 @@ const useForm = (initialValues: any, loginValidate: any) => {
                         setCookies('user', response.data, { path: '/' })
                         toast.success(("Login was successful. Welcome " + response.data.fname + " " + response.data.lname))
                         history.push("/list")
-                        console.log(response.data.id);
-                        dispatch(setId({ id: response.data.id }))
+                        // dispatch(setId({ id: response.data.id }))
```
Comment (GattieTime) [inline src/components/Account/LoginValidationHook.tsx:36]: why are we committing commented code?

File: src/components/Admin/Adminpage.tsx [hunk 2]
```diff
@@ -26,9 +26,9 @@ const Adminpage = () => {
     }
     const { data, isLoading } = useGetUserByIdQuery(userId);
     let users = data;
-    dispatch(setUsers(users))
+    dispatch(setUsers(users));
   }
-  SetAdminRedux();
+  // SetAdminRedux();
```
Comment (GattieTime) [inline src/components/Admin/Adminpage.tsx:31]: why comment?

File: src/components/Portfolio/PortfolioList/PortfolioList.tsx [hunk 3]
```diff
@@ -46,15 +46,15 @@ const PortfolioList = () => {
         console.log(response.data);
       })
       .catch((error) => {
-        console.log(error);
+        console.log("error");
```
Comment (GattieTime) [inline src/components/Portfolio/PortfolioList/PortfolioList.tsx:49]: Why?

File: src/components/Portfolio/PortfolioList/PortfolioTable.tsx [hunk 4]
```diff
@@ -36,7 +38,9 @@ function PortfolioTable(props: any) {
       .get(`${portfolioUrl}/${id}`)
       .then((response) => {
         setCookie("portfolio", response.data, { path: "/" });
-        window.location.pathname = pathname;
+        // <Link to={pathname} />
```
Comment (GattieTime) [inline src/components/Portfolio/PortfolioList/PortfolioTable.tsx:41]: commented code

File: src/features/FullPortfolioSlice.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { RootState } from '../store/Store';
+
+
+
+// export interface FullPortfolio {
```
Comment (GattieTime) [inline src/features/FullPortfolioSlice.tsx:6]: commented code.

File: src/features/FullPortfolioSlice.tsx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+// }
+
+export interface FullPortfolio {
+    fullPortfolio: {}
+
+}
+
+const initialState: FullPortfolio = {
+    fullPortfolio: {}
+    // portfolio: {
```
Comment (GattieTime) [inline src/features/FullPortfolioSlice.tsx:44]: commented code

File: src/features/PortfolioSlice.tsx [hunk 7]
```diff
@@ -0,0 +1,49 @@
+import Portfolio from '../interfaces/Portfolio';
+import User from '../interfaces/User';
+import { useSelector } from "react-redux";
+// import { useAppSelector } from '../store/Hooks';
```
Comment (GattieTime) [inline src/features/PortfolioSlice.tsx:4]: comment

File: src/features/PortfolioSlice.tsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+import { userState, } from '../features/UserSlice';
+// import { createSlice } from '@reduxjs/toolkit';
```
Comment (GattieTime) [inline src/features/PortfolioSlice.tsx:6]: comment

File: src/features/PortfolioSlice.tsx [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+    submitted: false,
+    approved: false,
+    reviewed: false,
+    feedback: "string",
+    admin: User1
+}
+
+const URL = "http://localhost:8081/api";
+
+// const portfolioSlice = createSlice({
```
Comment (GattieTime) [inline src/features/PortfolioSlice.tsx:30]: comment

File: src/features/UserSlice.tsx [hunk 10]
```diff
@@ -27,7 +27,7 @@ const initialState: userState = {
 }
 
 export const userSlice = createSlice({
-    name: "users",
+    name: "uss",
```
Comment (GattieTime) [inline src/features/UserSlice.tsx:30]: why?

File: src/index.tsx [hunk 11]
```diff
@@ -6,18 +6,24 @@ import reportWebVitals from './reportWebVitals';
 import { Auth0Provider } from '@auth0/auth0-react';
 import { Provider } from 'react-redux';
 import { store } from './store/Store';
+import { loadState, saveState } from './store/LocalStorage'
 import { Front } from 'react-bootstrap-icons';
 
+store.subscribe(() => {
```
Comment (GattieTime) [inline src/index.tsx:12]: Why are saving Redux Store to local storage?

File: src/store/LocalStorage.tsx [hunk 12]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (GattieTime) [inline src/store/LocalStorage.tsx:1]: Why are we using local storage?

**Review by GattieTime — CHANGES_REQUESTED** [review]

File: src/store/LocalStorage.tsx (hunk 12 above)
Comment (munibk22) [reply src/store/LocalStorage.tsx:1]: removed local storage, page refreshing was causing issues, but ended up fixing the issues, but did not remove local storage.

File: src/index.tsx (hunk 11 above)
Comment (munibk22) [reply src/index.tsx]: removed local storage, page refreshing was causing issues, but ended up fixing the issues, but did not remove local storage.

File: src/features/UserSlice.tsx (hunk 10 above)
Comment (munibk22) [reply src/features/UserSlice.tsx]: Looks like keyboard was accidently pressed, changed name back.

File: src/features/PortfolioSlice.tsx (hunk 9 above)
Comment (munibk22) [reply src/features/PortfolioSlice.tsx]: Removed commented code.

File: src/features/PortfolioSlice.tsx (hunk 8 above)
Comment (munibk22) [reply src/features/PortfolioSlice.tsx]: Removed commented code.

File: src/features/FullPortfolioSlice.tsx (hunk 5 above)
Comment (munibk22) [reply src/features/FullPortfolioSlice.tsx]: Removed commented code.

File: src/components/Portfolio/PortfolioList/PortfolioTable.tsx (hunk 4 above)
Comment (munibk22) [reply src/components/Portfolio/PortfolioList/PortfolioTable.tsx]: Removed commented code.

File: src/components/Portfolio/PortfolioList/PortfolioList.tsx (hunk 3 above)
Comment (munibk22) [reply src/components/Portfolio/PortfolioList/PortfolioList.tsx:47]: Removed  console.log(response.data);

File: src/components/Admin/Adminpage.tsx (hunk 2 above)
Comment (munibk22) [reply src/components/Admin/Adminpage.tsx]: Removed commented code. No longer needs to be called on page load.

File: src/components/Account/LoginValidationHook.tsx (hunk 1 above)
Comment (munibk22) [reply src/components/Account/LoginValidationHook.tsx]: Removed commented code.

File: src/components/Portfolio/PortfolioList/PortfolioList.tsx (hunk 3 above)
Comment (munibk22) [reply src/components/Portfolio/PortfolioList/PortfolioList.tsx]: removed quotes from error. Not sure why it was put in quotations.

**Review by GattieTime — APPROVED** [review]

**Pull request merged**